wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -27,8 +27,17 @@
                                 {{ message|safe }}
                             </li>
                         {% endfor %}
-                    </ul>
-                {% endif %}
+                    {% endif %}
+                </ul>
+                <template data-w-messages-target="template" data-type="success">
+                    <li class="success">{% icon name="success" classname="messages-icon" %}<span></span></li>
+                </template>
+                <template data-w-messages-target="template" data-type="success">
```

Comment:
This should be `data-type="error"` - PR up to fix - https://github.com/wagtail/wagtail/pull/10281

Reply:
I would definitely do a strict double check from now on, my bad :(

Reply:
I missed it also - this may have happened when I split to discrete templates. It's all good.